src/main/java/org/rumbledb/api/Item.java

```diff
@@ -261,6 +261,24 @@ public DateTime getDateTimeValue() {
         throw new RuntimeException("Item is not a DateTime.");
     }
 
+    /**
```

Comment:
Same comment as in the other PR: can we reuse getDateTimeValue() across all these items?